packages/account-export/INTEGRATION.md

```diff
@@ -0,0 +1,47 @@
+# @near-wallet-selector/account-export
+
+## Wallet Selector Integration
+
+Implementing account imports in Wallet Selector requires either:
+- Direct method call to an installed browser extension (injected wallets only)
+- URL with encrypted account data, either a link to a wallet domain or a deep link for mobile wallets
+
+### Injected Wallets
+For injected wallets implemented in browser extensions, calling a method directly on this plugin is permitted without encryption of account data.
+To use this method, an injected wallet would implement the [importAccountsInSecureContext](../core/src/lib/wallet/wallet.types.ts) method e.g.:
+```ts
+importAccountsInSecureContext(params: AccountImportSecureContextParams): Promise<void> {
+    params.forEach(({ accountId, privateKey }) => {
+        this.wallet.importAccount({ accountId, privateKey });
+    });
+}
+```
+
+Note that this is the default expected behavior for injected wallets. Injected wallets may opt into the URL-based approach described below by setting
+the `useUrlAccountImport` flag on `InjectedWalletMetadata` to `true`.
+
+### URL-Based
+
+Wallets may instead opt to implement a URL-based strategy, including browser wallets, mobile wallets via deep links, and injected wallets with a web component capable of
+communicating with an extension. To use a URL, the [buildImportAccountsUrl](../core/src/lib/wallet/wallet.types.ts) must be implemented to return the base URL for the account import. The encrypted data will
+then be appended to this URL as a fragment before redirecting the user to the composed URL, e.g.:
+```ts
+buildImportAccountsUrl(): string {
+  /* this URL must exist and parse the fragment data */
+  return `https://awesome-near-wallet.xyz/${this.state.networkId}`;
+}
+```
+
+The Wallet Selector will use this base URL and append encrypted data to it before redirecting there, so the final URL would be in the form of `"https://awesome-near-wallet.xyz/mainnet#xyz...abc"`
+
+:warning:
+**Once the user is redirected, the destination site should call `history.replaceState()` to prevent user data from being unnecessarily exposed in the browser.**
+:warning:
```

Comment:
this warning label here shown as:
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/6027014/214189654-0e6c0ac7-a4aa-48d2-807f-2daaece5701f.png">

is it mean to be `**warning**` ?

Reply:
`:warning:` just shows ⚠️, just something to grab attention. You can see it formatted here: https://github.com/near/wallet-selector/blob/e50370145ee05f3643f8d4488c47f2f09018678f/packages/account-export/INTEGRATION.md

Reply:
my apologies, thought markdowns works similarly on online readme editor tools..